src/components/Web3Status/index.tsx

```diff
@@ -124,7 +124,11 @@ function newTransactionsFirst(a: TransactionDetails, b: TransactionDetails) {
 }
 
 const SOCK = (
-  <span role="img" aria-label="has socks emoji" style={{ marginTop: -4, marginBottom: -4 }}>
+  <span
+    role="img"
+    aria-label={t({ id: 'wallet.hasSocks', message: 'has socks emoji' })}
```

Comment:
does this work? i think not because it's a constant, and not a function component

Reply:
that's why I'm using t():string here instead of <Trans>. t returns a string.

Reply:
i mean the SOCKS is a constant and is only evaluated at compile time so the label wouldnt change when the locale changes

Reply:
You're right.. it doesn't change. I'll revert this until I figure out a better solution.

Reply:
just make it a function component instead of a const

Reply:
I was saying, instead of doing `const SOCK = (<jsx.../>)`

do 
```
const Sock = function Sock() {
  return <jsx.../>
}
```

This will fix the translation issue